upstairs/src/client.rs

```diff
@@ -2334,6 +2340,31 @@ enum NegotiationState {
     Done,
 }
 
+/// Result value from [`DownstairsClient::enqueue`]
+pub(crate) enum EnqueueResult {
+    /// The given job should be marked as in progress and sent
+    Send,
+
+    /// The given job should be marked as in progress, but not sent
+    ///
+    /// This is used when the Downstairs is Offline; we want to mark the job as
+    /// in-progress so that it's eligible for replay, but the job should not
+    /// actually go out on the wire.
+    Drop,
```

Comment:
What about the name `Hold` instead of `Drop`?

Reply:
Much better, changed!